lopezMidterm/src/components/LightSwitch.jsx

```diff
@@ -0,0 +1,26 @@
+import  { useState } from 'react';
+
+function LightSwitch() {
+  const [isDarkMode, setIsDarkMode] = useState(false);
+
+  const handleToggle = () => {
+    setIsDarkMode(!isDarkMode);
+    document.body.classList.toggle('dark-mode', !isDarkMode);
+    document.body.classList.toggle('light-mode', isDarkMode);
```

Comment:
While this works, this is not what we covered in class. You must leverage the Context API to implement dark mode which starts here https://neit.instructure.com/courses/56465/pages/live-coding-dark-mode?module_item_id=2525730